doc/development.rst

```diff
@@ -28,7 +28,7 @@ __ fork_
 Documentation
 -------------
 
-This is the documentation you are reading right now. The raw files can be found in ``doc/``. The HTML version user documentation is build with `Sphinx`_. To rebuild this documentation install Sphinx with ``pip install sphinx sphinx-bootstrap-theme`` and call ``python setup.py build_sphinx``. The output appears in ``build/sphinx/html``. The latest documentation can be found at ``http://mapproxy.org/docs/lates/``.
+This is the documentation you are reading right now. The raw files can be found in ``doc/``. The HTML version user documentation is build with `Sphinx`_. To rebuild this documentation install Sphinx with ``pip install sphinx sphinx-bootstrap-theme`` and call ``sphinx-build doc/ docs``. The output appears in ``docs``. The latest documentation can be found at ``https://mapproxy.github.io/mapproxy/latest/``.
```

Comment:
``python setup.py build_sphinx`` was introduced [here](https://github.com/mapproxy/mapproxy/blame/fe53a726dcb0ba37cced76e4b1a6cd273b548967/release.py#L57C9-L57C37) I believe.  It has been removed since then (see ``release.py`` section of [this commit](https://github.com/mapproxy/mapproxy/commit/9965ccfe01e0e68e691f337dda1dd354832c59a8)). Now ``sphinx-build doc/ docs`` is used in [documentation build](https://github.com/mapproxy/mapproxy/blob/c792f4c6d74175262b573e30259bf7ca23602978/.github/workflows/ghpages.yml#L25C12-L25C34).